config/dws/nnf-ruleset.yaml

```diff
@@ -97,13 +97,15 @@ spec:
     ruleDefs:
       - key: "^name$"
         type: "string"
+        pattern: "^[a-z][a-z0-9-]+$"
         isRequired: true
         isValueRequired: true
       - key: "^profile$"
         type: "string"
+        pattern: "^[a-z][a-z0-9-]+$"
         isRequired: true
         isValueRequired: true
-      - key: '^(DW_JOB_|DW_PERSISTENT_)[A-Za-z][A-Za-z0-9-]+$'
+      - key: '^(DW_JOB_|DW_PERSISTENT_)[a-z][a-z0-9_]+$'
```

Comment:
Did you mean you swap the hyphen for underscores here?

Reply:
Yes, environment variables cannot have dashes--try it in your shell.